bench/pytorch.py

```diff
@@ -0,0 +1,130 @@
+import timeit
+
+
+# DeepONet
+
+import_code = """import deepxde as dde
+import torch
+"""
+
+n_iters = 100
+fwd_timed_arr = [0.0] * 5
+training_timed_arr = [0.0] * 5
+
+for i in range(1, 6):
+    setup_code = f"""eval_points = 128
+batch_size = 64
+dim_y = 1
+m = 32
+
+ch_branch = [m] + [64]*{i} + [128]
+ch_trunk = [dim_y] + [64]*{i} + [128]
+u = torch.rand(batch_size, m)
+y = torch.rand(batch_size, dim_y)
+output = torch.rand(batch_size, dim_y)
+
+u_ = torch.rand(8, m)
+y_ = torch.rand(8, dim_y)
+output_ = torch.rand(8, dim_y)
+
+data = dde.data.Triple(
+    X_train= (u,y), y_train= output, X_test=(u_, y_), y_test= output_
+)
```

Comment:
The dimensions are wrong here.

The julia version has sizes:

```julia
julia> size(u)
(32, 64)

julia> size(y)
(1, 128, 64)
```

while the pytorch versions have `(64, 32)` and `(64, 1)`